### ISSUE TITLE
GUI Color List

### ISSUE THREAD

**Issue by GitsCpmb** [hexchat/hexchat#2313]
So I've just tried Hexchat and was playing with the interface trying to change some of the colors. I've found themes and loaded some, and have no idea what colors do what.

I've read the colors.conf and it tells me NOTHING about what color numbers affect what elements.

Is is possible for someone to post a list in the help docs of the color numbers and elements, and maybe even include these in the code that generates the colors.conf.

For example

color_0     chat window background
color_1     chat window user text
color_2     server window active user
color_3     server window away user

etc......

Pretty much flying blind with zero info available.

Much thanks.

**Comment by TingPing** [comment]
No such thing is possible, themes are split into two parts: a palette and event formatting.

https://hexchat.readthedocs.io/en/latest/appearance.html#theming

**Comment by GitsCpmb** [comment]
Ok. I took a look at pevents.conf and see what you mean.

So then is there a comprehensive list of events available, or are those listed the only events recognized by the software.

Essentially, I'd like to change the default color of the users list alone. I've discovered this can be done by changing the foreground color. But that affects all server messages (it seems), and that's not what I'm hoping to achieve.

**Comment by TingPing** [comment]
> So then is there a comprehensive list of events available, or are those listed the only events recognized by the software.

The full list is always in *Settings > Text Events* along with previews and descriptions.

**Comment by GitsCpmb** [comment]
Ah. Very cool. Thank you muchly. Will check it out :)

**Comment by GitsCpmb** [comment]
I've looked at the events lists and found something interesting.

Given that the user list is in a seperate window, it must query the server using the USERLIST command. I issued the USERLIST command and got a whole list of info. So I'm certain the userlist window filters this info, and then lists provided usernames it in it's own window. There is probably a  specific function handling the userlist query data and the user state change messages from the server.

It must be possible to create an event ("Server User List") and a color setting specifically for that function to fetch, such as an "Active User" color icon on the pallete in addition to the "Away User" color icon.

It would just be a matter of adding an feature to a  function that already does the majority of the same work.
